packages/x-components/src/x-modules/experience-controls/components/experience-controls.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import Vue from 'vue';
+  import { XOn } from '../../../components/decorators/bus.decorators';
+  import { XEvent, XEventsTypes } from '../../../wiring/events.types';
+
+  export default class ExperienceControls extends Vue {
+    /**
+     * This component hear to the change of events in the {@link XStoreModule}, receive them, iterate them and fire them.
+     *
+     * @param events - The list of the events on the module.
+     */
+    @XOn('ExperienceControlsEventsChanged')
+    onEventsChanged(events: Partial<XEventsTypes>): void {
+      Object.entries(events).forEach(([eventName, eventPayload]) => {
+        this.$x.emit(eventName as XEvent, eventPayload);
+      });
+    }
+  }
+</script>
```

Comment:
You're missing the render function.
A renderless component without a `<template>` still needs to define in what way it's "rendered"

```suggestion

    // eslint-disable-next-line @typescript-eslint/no-empty-function
    render(): void {}
  }
</script>
```